ext/standard/tests/bug34834.phpt

```diff
@@ -0,0 +1,40 @@
+--TEST--
+BUG #34834 (array_merge_recursive() merges arrays with objects with arrays)
+--CREDITS--
+Rodrigo Prado de Jesus <royopa [at] gmail [dot] com>
+User Group: PHPSP #PHPTestFestBrasil
+--FILE--
+<?php
+class A 
+{ 
+  var $field = array(1);
+}
+
+$a = new A;
+$x = array("a" => $a);
+
+$y = array("a" => array("field" => array(2)));
+
+var_dump(array_merge_recursive($x,$y));
+?>
+--XFAIL--
```

Comment:
We won't accept `XFAIL` tests as we should reopen the Bug or provide a solution.